.ci/setup_env_debian.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || source "/usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+echo "Enable librbd1 repository"
+sudo bash -c "cat <<EOF > /etc/apt/sources.list.d/unstable.list
+deb http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
+deb-src http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
+EOF"
+
+echo "Lower priority than stable"
+sudo bash -c "cat <<EOF > /etc/apt/preferences.d/unstable
+Package: *
+Pin: release a=unstable
+Pin-Priority: 10
+EOF"
+
+echo "Update apt repositories"
+sudo -E apt update
+
+echo "Install chronic"
+sudo -E apt install -y moreutils
+
+echo "Install curl"
+sudo -E apt install -y curl
+
+echo "Install librbd1"
+chronic sudo -E apt install -y -t unstable librbd1
+
+echo "Install kata containers dependencies"
+chronic sudo -E apt install -y libtool automake autotools-dev autoconf bc alien libpixman-1-dev coreutils
+
+echo "Install qemu dependencies"
+chronic sudo -E apt install -y libcap-dev libattr1-dev libcap-ng-dev librbd-dev
+
+echo "Install kernel dependencies"
+chronic sudo -E apt install -y libelf-dev
+
+echo "Install CRI-O dependencies for Debian"
+chronic sudo -E apt install -y libglib2.0-dev libseccomp-dev libapparmor-dev \
+        libgpgme11-dev go-md2man thin-provisioning-tools
+
+echo "Install bison binary"
+chronic sudo -E apt install -y bison
+
+echo "Install libudev-dev"
+chronic sudo -E apt-get install -y libudev-dev
+
+echo "Install Build Tools"
+chronic sudo -E apt install -y build-essential python pkg-config zlib1g-dev
+
+echo -e "Install CRI-O dependencies available for Debian"
+chronic sudo -E apt install -y libdevmapper-dev btrfs-tools util-linux
+
+chronic sudo -E apt install -y libostree-dev
+
+echo "Install YAML validator"
+chronic sudo -E apt install -y yamllint
+
+echo "Install tools for metrics tests"
+chronic sudo -E apt install -y smem jq
+
+if [ "$(arch)" == "x86_64" ]; then
+	echo "Install Kata Containers OBS repository"
+	obs_url="http://download.opensuse.org/repositories/home:/katacontainers:/release/xUbuntu_17.10"
```

Comment:
in general this pr l g t m, my only question is why 17.10?

Reply:
@devimc , currently we do not have OBS packages for debian https://github.com/kata-containers/packaging/issues/247, however, the OBS packages for ubuntu are working perfectly in debian https://github.com/kata-containers/documentation/pull/306. This is a temporary solution until is done https://github.com/kata-containers/packaging/issues/247.

Reply:
How about we make this PR "dnm" until https://github.com/kata-containers/documentation/pull/306 lands to avoid having to use the Ubuntu packages for Debian?

Reply:
@jodh-intel and @devimc changes applied